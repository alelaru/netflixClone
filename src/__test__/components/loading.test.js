import { render } from "@testing-library/react";
import Loading from "../../components/loading";


describe("<Loading/>", () => {

    it("Renders the <Loading/> with populated data", () => {

        const {container, getByTestId} = render(
            <Loading src={"images/karl.png"} data-testid="loading"></Loading  > 
        ) 
        expect(getByTestId("loading")).toBeTruthy();
        expect(getByTestId("loading-picture")).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();
    })

    it("Renders the <Loading ReleaseBody/> with populated data", () => {

        const {container} = render(
            <Loading.ReleaseBody></Loading.ReleaseBody>
        ) 
        expect(container.firstChild).toMatchSnapshot();
    })


})