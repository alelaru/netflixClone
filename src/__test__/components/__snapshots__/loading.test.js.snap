// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> Renders the <Loading ReleaseBody/> with populated data 1`] = `null`;

exports[`<Loading/> Renders the <Loading/> with populated data 1`] = `
<div
  class="sc-bdnxRM Krhwz"
  data-testid="loading"
>
  <img
    class="sc-gtsrHT ikuiYV"
    data-testid="loading-picture"
    src="/images/users/images/karl.png.png"
  />
</div>
`;
